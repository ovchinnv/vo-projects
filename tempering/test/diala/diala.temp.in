
 output {
  minwarnlev=-1
 }

 tempering {
 
 }