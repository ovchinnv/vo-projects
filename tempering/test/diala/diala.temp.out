 MESSAGE (tempering_init_from_acemd): INITIALIZING TEMPERING PLUGIN
